internalsdk/common/ad.go

```diff
@@ -0,0 +1,15 @@
+package common
+
+// AdSettings is an interface for retrieving mobile ad settings from the global config
+type AdSettings interface {
```

Comment:
Can't we use the same on `android.go`?

Reply:
I moved it temporarily to common to avoid an import cycle in internalsdk with the mocks code, but that should be fixed now.